src/tests/test_pyproxy.py

```diff
@@ -62,6 +62,49 @@ def get_value(self, value):
     assert selenium.run_js("return pyodide.pyimport('f').toString()").startswith("<Foo")
 
 
+def test_pyproxy2(selenium):
+    result = selenium.run_js(
+        """
+        pyodide.runPython(`
+            class Foo:
+                pass
+            f = Foo()
+        `);
+        return pyodide.pyimport('f').toString();
+        """
+    )
+    print(result)
+    assert result.startswith("<Foo")
+
+
+def test_pyproxy3(selenium):
+    selenium.run_js(
+        """
+        window.jspython = pyodide.pyimport("open");
+        pyodide.runPython(`
+            from js import jspython
+            jspython is open
```

Comment:
```suggestion
            assert jspython is open
```
?
Is this related to this PR though? Maybe it would be better to add theses tests in a separate PR?

Reply:
Again this is a minimization of an existing test that was breaking and should be removed in any case (it did lead to PR #1098).